### PR TITLE
C# fizzbuzz method 2 implementation issue #41

### PR DESCRIPTION
**Pull request by AndreiG23**
A different implemantion of FizzBuzz in C#.

### CODE REVIEW

File: Go/FizzBuzz.go [hunk 1]
```diff
@@ -3,17 +3,16 @@ package main
...
-		} else if i%5 == 0 {
-			fmt.Printf("Buzz, ")
-		} else if i%3 == 0 {
-			fmt.Printf("Fizz, ")
-		} else {
-			fmt.Printf("%d, ", i)
-		}
-	}
-}
+        for i := 1; i < 101; i++ {
```
Comment (ashish-r) [inline Go/FizzBuzz.go]: different file name

File: Ruby/FizzBuzz.rb [hunk 2]
```diff
@@ -1 +1,13 @@
-100.times{|index| puts (index+1)%15==0?"FizzBuzz":(index+1)%3==0?"Fizz":(index+1)%5==0?"Buzz":index+1}
+#!/usr/bin/env ruby -w
```
Comment (ashish-r) [inline Ruby/FizzBuzz.rb]: different file name

**Review by ashish-r — CHANGES_REQUESTED** [review]
Hi, You need to submit your Go and Ruby solutions in a different file name. Your solutions are overwriting the already submitted solutions.

File: Ruby/FizzBuzz.rb (hunk 2 above)
Comment (AndreiG23) [reply Ruby/FizzBuzz.rb]: I didn't modify them. They were already modified when I forked the project.

**Comment by AndreiG23** [comment]
Now it's ok? I don't know why the go and ruby files got modified. I didn't touch any of these files.

**Comment by ashish-r** [comment]
@AndreiG23 Thanks for raising the issue. I have now fixed the issue. You can create a new PR or reset all unwanted modified files and then commit. For details check https://stackoverflow.com/questions/2733873/reverting-a-single-file-to-a-previous-version-in-git

**Pull request closed without merging**